recipes/densmap-learn/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "densmap-learn" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 6e9d8a2286335633b924fbb631a0f4fbbb1b6c07714feaae86ec3fc0e9124716
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
```

Comment:
Under `host` you should only need `pip` and `python` since the others do not appear to be imported directly or indirectly by the `setup.py` for the project.